### ISSUE TITLE
Can ISD1820 (a recording module) be used to record my voice message and saving it as a wave file in SD card?

### ISSUE THREAD

**Issue by mdshabaz002** [TMRh20/TMRpcm#119]
The recording module I'm using has four important pins. 2 pins goes for supply and ground. There is a recording pin that i have connected to A0. There is also a play pin which I guess I'm letting out!
Its a novice issue, as I'm new to it. Any help or suggestion would be appreciated.

**Comment by LDM30** [comment]
No, the data the microphone records has no way of reaching the Arduino.
